yaag/models/apicall.go

```diff
@@ -0,0 +1,19 @@
+package models
+
+type ApiCall struct {
+	Id int
```

Comment:
I've kept them as it is for one reason:

If in the future we allow an editor to edit the source file behind the documentation, Camel case would make more sense than snake case.